.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ defaults:
     # reference syntax)
     working_directory: ~/coq
     docker:
-      - image: coqci/base:V2018-05-07-V2
+      - image: registry.gitlab.com/coq/coq:bionic_coq-V2018-05-15-V1
```

Comment:
That seems like it should be its own PR.

Reply:
Or just remove circle.

Reply:
Moved to its own PR.